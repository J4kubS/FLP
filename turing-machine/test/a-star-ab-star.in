S a A R
S a B R
A a A R
A   F R
B b C R
C a B R
C   F R
abababababab
