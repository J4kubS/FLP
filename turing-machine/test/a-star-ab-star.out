Sabababababab
aBbababababab
abCababababab
abaBbabababab
ababCabababab
ababaBbababab
abababCababab
abababaBbabab
ababababCabab
ababababaBbab
abababababCab
abababababaBb
ababababababC
abababababab F
